templates/CTCORECHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## CleverTap Android SDK CHANGE LOG
 
+### Version 4.0.2 (December 28, 2020)
```

Comment:
@kumaratulismu You need to run the Gradle task to generate the CHANGELOG in the docs folder after making changes to the template folder.